source/backend/database/models/recipesModel.js

```diff
@@ -46,6 +46,17 @@ async function createRecipe(payload, ingredientsArray) {
   });
 }
 
+/**
+ * update recipe by recipe id
+ * @param {*} payload 
+ * @param {*} recipeId 
+ */
+async function updateRecipe(payload, recipeId) {
+  await db('recipes')
+    .update(payload)
+    .where({recipeId});
+}
+
```

Comment:
This looks good, but I think we need to do a transaction to update the ingredients table as well and potentially the recipeIngredients table (if there are extra/new ingredients)